docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -1,280 +1,366 @@
+
+At this stage you should see number of pods running. Please confirm that all pods are healthy by running CLI command `$ oc get pods -n pelorus` to continue with this tutorial. If there is any pod in `Error` or `CrashLoopBackOff` state, this tutorial may fail at any further step:
+
+```shell
+$ oc get pods -n pelorus
+NAME                                                   READY   STATUS      RESTARTS        AGE
+committime-exporter-1-7zvmp                            1/1     Running     0               2m10s
+committime-exporter-1-deploy                           0/1     Completed   0               2m13s
+deploytime-exporter-1-deploy                           0/1     Completed   0               2m13s
+deploytime-exporter-1-hrnqq                            1/1     Running     0               2m10s
+failure-exporter-1-deploy                              0/1     Completed   0               2m13s
+failure-exporter-1-r55lp                               1/1     Running     0               2m10s
+grafana-deployment-65765d5594-9vmqx                    2/2     Running     0               2m14s
+grafana-operator-controller-manager-bf759cf9b-4gxxl    2/2     Running     0               2m58s
+pelorus-operator-controller-manager-8455c65d76-b5z7k   1/1     Running     0               2m45s
+prometheus-operator-6fdc9c5494-ggwwk                   1/1     Running     0               2m49s
+prometheus-prometheus-pelorus-0                        3/3     Running     1 (2m10s ago)   2m14s
+prometheus-prometheus-pelorus-1                        3/3     Running     1 (2m9s ago)    2m14s
 ```
 
-Apply the updated values for Pelorus by executing:
+#### Accessing Pelorus dashboard
+
+To view the Pelorus measurements we need to gain access to its' Grafana dasboard. To get the URL of the Grafana dashboard use the following command:
+```shell
+$ oc get route grafana-route --namespace pelorus -o=go-template='https://{{.spec.host | printf "%s\n" }}'
 ```
-helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+
+In the Grafana dasboard navigate to **search** and select **"pelorus ➔ Software Delivery Performance - By App"**. An empty dashboard without any measurements should be presented. That is expected, because we did not deploy our sample application yet!
+
+![pelorus_dashboard_no_data](../img/pelorus-dashboard-no-data.png)
+
+#### Congratulations
+
+Pelorus is now configured to measure the sample application.
+
+It is time to deploy the sample application to view measurements from Pelorus' Grafana dashboard. Plase keep the Grafana page open.
+
+
+## Sample Aplication Demo
+
+### Lead Time for Change and Deployment Frequency
+
+In this section you see how Pelorus measures two DORA metrics: **Lead time for change** and **Deployment frequency**.
+
+![dora_lead_deployment](../img/dora_metrics1.png)
+
+#### Install the sample application
+
+Clone **your** forked copy from the [GitHub project fork](#github-project-fork) step:
+
+```shell
```

Comment:
this should be a simple git clone command, no need to complex the command

Reply:
Because we have commands in two places I wanted to make it copy-paste, so:

```shell
# Adjust <your_org> to your GitHub fork org:
$ export GITHUB_FORK_ORG=<your_org>

$ git clone git@github.com:${GITHUB_FORK_ORG}/mig-demo-apps.git
```

and

```shell
# Adjust <your_org> to your GitHub fork org:
$ export GITHUB_FORK_ORG=<your_org>

$ cd mig-demo-apps/apps/todolist-mongo-go
$ sed -i "s/your_org/${GITHUB_FORK_ORG}/g" mongo-persistent.yaml
$ oc create -f mongo-persistent.yaml
```

should work independently as copy-paste commands. I am a bit worried that user may not notice all the places where inside the code <your_org> requires replacement that is why it's outlined as it is.

We could have one code snippet like:

From your forked GitHub repository install the `todolist-mongo-go` sample application in your OpenShift cluster:

```shell
# Adjust <your_org> to your GitHub fork org:
$ export GITHUB_FORK_ORG=<your_org>

$ git clone git@github.com:${GITHUB_FORK_ORG}/mig-demo-apps.git
$ cd mig-demo-apps/apps/todolist-mongo-go
$ sed -i "s/your_org/${GITHUB_FORK_ORG}/g" mongo-persistent.yaml
$ oc create -f mongo-persistent.yaml
```

Reply:
but because of the `$`s it is not copy paste

But would not change this now

Reply:
@mateusoliveira43 
ok, so closing.

The "$" at the beginning is to really mimic the prompt of the shell and easy reading output from the code snippet. This way we can easily distinguish between the command and the output from the command, e.g.:

```shell
$ echo echo
echo
```